Core/CategoryComparer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Core.Categories;
+using System.Collections.Generic;
+
+namespace Core
+{
+    internal class CategoryComparer : IEqualityComparer<ICategory>
+    {
+        public bool Equals(ICategory x, ICategory y)
+        {
+            if (ReferenceEquals(x, y)) 
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6003477)